gulpjs/cloudbuild.yaml

```diff
@@ -0,0 +1,32 @@
+# In this directory, run the following command to build this builder.
+# $ gcloud container builds submit . --config=cloudbuild.yaml
+
+steps:
+# Step 0
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['build', '--tag=gcr.io/$PROJECT_ID/gulpjs', '.']
+  dir: 'gulpjs'
```

Comment:
this file is meant to be executed from this directory, not the top of the repository (omit "gulpjs" from the `dir` fields).